.github/workflows/android_deploy_staging.yml

```diff
@@ -49,10 +49,15 @@ jobs:
       - name: Build Android apk
         run: flutter build apk --flavor staging --debug --build-number $GITHUB_RUN_NUMBER
 
+      - name: Get PR information
+        uses: 8BitJonny/gh-get-current-pr@2.2.0
+        id: PR
```

Comment:
To generate release notes for merged PRs on the `develop` branch, we need to determine a range of `fromTag` to `toTag` (both tag name or commit sha supported):
- fromTag: it uses the last tag if skipped.
- toTag: should be the HEAD commit.
- To generate only the latest merged PR on the `develop` branch, use `max_pull_requests = 1` config. However, to reuse only one `changelog-config.json` file for all workflows, we can edit it before generating.

```suggestion
      - name: Find HEAD commit
        id: head
        run: echo "sha=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT

      - name: Limit changelog to the latest pull request only
        uses: jossef/action-set-json-field@v2
        with:
          file: .github/workflows/changelog-config.json
          field: max_pull_requests
          value: 1

      - name: Build changelog on "develop"
        id: changelog
        uses: mikepenz/release-changelog-builder-action@v4
        with:
          configuration: ".github/workflows/changelog-config.json"
          # Listing PRs from the last tag to the HEAD commit
          toTag: ${{ steps.head.outputs.sha }}
          token: ${{ secrets.GITHUB_TOKEN }}
```

Tested successfully here https://github.com/nimblehq/flutter-templates/actions/runs/6747818441/job/18344811327

Test branch https://github.com/nimblehq/flutter-templates/compare/chore/198-test-merge-1...test-lucas?expand=1

```
## Others

- [TEST] Ignore please
   - PR: #279
```